### PR TITLE
ignore-missing-py-typed

### PR DESCRIPTION
**Pull request by KotlinIsland**
image

### CODE REVIEW

File: test-data/unit/cmdline-based.test [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+[file mypy.ini]
+\[mypy]
+\[mypy-a]
+ignore_missing_py_typed = true
+[file pypath/a/__init__.py]
+x = 0
+[file pypath/b/__init__.py]
+y = 0
+[out]
+main.py:2:1: error: Skipping analyzing "b": module is installed, but missing library stubs or py.typed marker  [import]
```
Comment (DetachHead) [inline test-data/unit/cmdline-based.test:15]: How do the other tests work? Wouldn't you have to specify a `py.typed` for every single other test?

File: test-data/unit/cmdline-based.test [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+[file main.py]
+from a import x
+reveal_type(x)
+[file pypath/a/__init__.py]
+x = 0
+[file pypath/a-stubs/__init__.pyi]
+x: str
+[out]
+main.py:2:13: note: Revealed type is "str"
+== Return code: 0
```
Comment (DetachHead) [inline test-data/unit/cmdline-based.test:30]: Add a test for when there's a `py.typed` file?

File: test-data/unit/cmdline-based.test (hunk 1 above)
Comment (KotlinIsland) [reply test-data/unit/cmdline-based.test:15]: mypy_path vs python_path

File: test-data/unit/cmdline-based.test (hunk 2 above)
Comment (KotlinIsland) [reply test-data/unit/cmdline-based.test:30]: added

**Review by DetachHead — APPROVED** [review]

**Pull request merged**